sqlglot/dialects/snowflake.py

```diff
@@ -48,6 +53,19 @@ def _snowflake_to_timestamp(args):
     return exp.UnixToTime.from_arg_list(args)
 
 
+def _unix_to_time(self, expression):
+    scale = expression.text("scale")
```

Comment:
really sorry about this. my recommendation was bad, i'm doing too many things at once.

i think how you had it was fine because scale is an expression, can you revert it back to expression.args.get("scale")

Reply:
fixed